src/Pos/Wallet/Web/Api.hs

```diff
@@ -6,18 +6,33 @@
 module Pos.Wallet.Web.Api
        ( WalletApi
        , walletApi
+       , WalletApiWithDocs
+       , walletApiWithDocs
+       , swaggerSpecForWalletApi
        ) where
 
 import           Data.Proxy                 (Proxy (Proxy))
```

Comment:
Btw, since you modify this file, you can remove this import (it's exported from `Universum`).